bin/package-update.sh

```diff
@@ -31,6 +31,8 @@ output 3 "Updating package textdomains..."
 # Replace text domains within packages with woocommerce
 find ./packages/woocommerce-blocks \( -iname '*.php' -o -iname '*.js' \) -exec sed -i.bak -e "s/'woo-gutenberg-products-block'/'woocommerce'/g" -e "s/\"woo-gutenberg-products-block\"/'woocommerce'/g" {} \;
 find ./packages/woocommerce-rest-api -iname '*.php' -exec sed -i.bak -e "s/, 'woocommerce-rest-api'/, 'woocommerce'/g" {} \;
+find ./packages/woocommerce-admin -iname '*.php' -exec sed -i.bak -e "s/, 'woocommerce-admin'/, 'woocommerce'/g" {} \;
+find ./packages/woocommerce-admin -iname '*.js' -exec sed -i.bak -e "s/, 'woocommerce-admin'/, 'woocommerce'/g" {} \;
```

Comment:
We need a better script for it, it's easy now to override important data with this, since those may be slugs for another functionality.

Reply:
@claudiosanches wouldn't this be true for the two lines above also? Do you have any ideas for a better approach?

Reply:
@timmyc I have a solution for it already: https://github.com/woocommerce/woocommerce/pull/25536